example-client/client.py

```diff
@@ -24,26 +47,44 @@ def run_structure(input: str) -> Optional[str]:
     Returns:
         The output of the Structure program.
     """
-    # Start a run with the args "What is 5 ^ 2".
+    # Start a run with the input args.
     # These args will be passed into our Structure program as standard input.
-    structure_run = create_structure_run(GT_STRUCTURE_ID, {}, [input])
+    structure_run = create_structure_run(
+        host=HOST,
+        api_key=GT_API_KEY,
+        structure_id=GT_STRUCTURE_ID, 
+        env={}, 
+        args=[input]
+        )
```

Comment:
Kind of strange formatting, can you run this through a VsCode format?

Reply:
something was FUBAR with Black